packages/frontend/src/components/NavBar/ExploreMenu.tsx

```diff
@@ -95,9 +91,7 @@ const ExploreMenu: FC<Props> = memo(({ projectUuid }) => {
                                 component={Link}
                                 title="Query using SQL runner"
                                 description="Access your database to run ad-hoc queries."
-                                to={`/projects/${projectUuid}/${
```

Comment:
is it still possible to access the old sql runner ?

Reply:
yup! https://lightdash-pr-11381.onrender.com/projects/3675b69e-8324-4110-bdca-059031aa8da3/sqlRunner